app/api/logging/config.py

```diff
@@ -1,9 +1,16 @@
 import logging
-import os
 import sys
 
 import api.logging.formatters as formatters
 import api.logging.pii as pii
+from api.util.env_config import PydanticBaseEnvConfig
+
+logger = logging.getLogger(__name__)
+
+
+class LoggingConfig(PydanticBaseEnvConfig):
+    log_format: str = "json"
+    log_level: int = logging.INFO
```

Comment:
Would it be easier for the user to make this a str instead? `setLevel()` accepts a string too. It will raise an error if it's invalid so that would happen in `configure_logging()`.

```python
>>> import logging
>>> logging.root.setLevel("INFO")
>>> logging.root.setLevel("DEBUG")
>>> logging.root.setLevel("NOTE")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.9/logging/__init__.py", line 1417, in setLevel
    self.level = _checkLevel(level)
  File "/usr/lib/python3.9/logging/__init__.py", line 194, in _checkLevel
    raise ValueError("Unknown level: %r" % level)
ValueError: Unknown level: 'NOTE'
```

Reply:
I see, so do something like `log_level: str = "INFO"`? Sure that does seem easier than the int version.